web/auth0.go

```diff
@@ -23,30 +23,43 @@ import (
 	"go.viam.com/utils"
 )
 
-// Auth0Config config for auth0.
-type Auth0Config struct {
+// AuthProviderConfig config options with constants that will probably need to be manually configured after
```

Comment:
I didn't want to rename as part of this PR as it would destroy the commit history on the squash and merge. I think I should do a `git mv` as a follow up